Guides.md

```diff
@@ -17,7 +27,7 @@ If you are using Windows you will have to download a ssh client such as [KiTTY](
 
 ## Access your files from home
 
-### Linux and BSD
+#### Linux and BSD
```

Comment:
Ser mindre kladdigt ut (pers pref).

> 5 juni 2016 kl. 00:53 skrev Rovanion Luckey <notifications@github.com>:
> 
> In Guides.md:
> 
> > @@ -17,7 +27,7 @@ If you are using Windows you will have to download a ssh client such as [KiTTY](
> >  
> >  ## Access your files from home
> >  
> > -### Linux and BSD
> > +#### Linux and BSD
> Vadan denna ändring?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>